source/uwp/AdaptiveCardTestApp/Expected/SourceCards/v1.2.Elements.Action.OpenUrl.Sentiment.d02e0c3.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Can you run the tests again and confirm that you've accepted all the necessary updates? I'd expect the images to change since you've changed the Action titles, and also the roundtripped ToJson files.